### PR TITLE
DXP-1110 Updated to new mesh definition

### PR DESCRIPTION
**Pull request by michalcukierman**
## 📋 Type of the Changes

- [ ] Breaking change
- [x] Non-breaking change
- [ ] Bug fix / minor change

## 🛠 Changes being made
Updated to new mesh definition
Disabled observability modules in e2e tests

## ✅ Checklist

- [x] My code follows the [code standards](https://github.com/streamx-dev/streamx/blob/main/CONTRIBUTING.md) of this project
- [x] Changed code is covered with unit tests
- [x] I have updated READMEs and java docs (if applicable)

### CODE REVIEW

File: e2e-tests/configs/overridden-nginx-conf [hunk 1]
```diff
@@ -0,0 +1,10 @@
+server {
+    listen       80;
+    listen  [::]:80;
+    server_name  localhost;
+
+    location /overridden {
```
Comment (michalcukierman) [inline e2e-tests/configs/overridden-nginx-conf:6]: Changed the config, to actually change something.

File: e2e-tests/src/main/java/dev/streamx/cli/StreamxTerminalCommandProducer.java [hunk 2]
```diff
@@ -25,7 +25,11 @@ public StreamxTerminalCommand produce() {
   }
 
   public String built() {
-    return "java -jar " + findStreamxJar();
+    return "java %s -jar %s".formatted(createParameters(), findStreamxJar());
+  }
+
+  private static String createParameters() {
+    return "-Dstreamx.runner.observability.enabled=false";
```
Comment (michalcukierman) [inline e2e-tests/src/main/java/dev/streamx/cli/StreamxTerminalCommandProducer.java:32]: Disabling observability in e2e, while it's not used

File: e2e-tests/src/test/java/dev/streamx/cli/StreamxCliPublicationIT.java [hunk 3]
```diff
@@ -25,9 +25,9 @@
 public class StreamxCliPublicationIT {
 
   private static final int CLI_SHORT_TIMEOUT_IN_SEC = 2;
-  @ConfigProperty(name = "streamx.cli.e2e.web.delivery.port.url", defaultValue = "http://localhost:8087/")
+  @ConfigProperty(name = "streamx.cli.e2e.web.delivery.url", defaultValue = "http://localhost:8087/")
   String webDeliveryPortUrl;
-  @ConfigProperty(name = "streamx.cli.e2e.nginx.port.url", defaultValue = "http://localhost:8089/")
+  @ConfigProperty(name = "streamx.cli.e2e.nginx.url", defaultValue = "http://localhost:8089/overridden/")
```
Comment (michalcukierman) [inline e2e-tests/src/test/java/dev/streamx/cli/StreamxCliPublicationIT.java:30]: Removed .port. because it's actually an URL

Comment (michalcukierman) [reply e2e-tests/src/test/java/dev/streamx/cli/StreamxCliPublicationIT.java:30]: Added the path added in custom configuration

File: e2e-tests/src/test/resources/filtered/mesh.yaml [hunk 4]
```diff
@@ -38,6 +38,7 @@ delivery:
         ports:
           - 8089:80
         repositoryVolume: "/usr/share/nginx/html"
-        volumes:
-          - "${project.basedir}/target/test-classes/overridden-nginx-conf:/etc/nginx/conf.d/default.conf"
+        volumesFrom:
+          configs:
+          - "overridden-nginx-conf:/etc/nginx/conf.d/default.conf"
```
Comment (michalcukierman) [inline e2e-tests/src/test/resources/filtered/mesh.yaml:43]: changed the file location to match the new convention (sources/secrets)

File: pom.xml [hunk 5]
```diff
@@ -39,7 +39,7 @@
     <quarkus.platform.group-id>io.quarkus.platform</quarkus.platform.group-id>
     <quarkus.platform.version>3.13.3</quarkus.platform.version>
 
-    <streamx.version>0.0.46</streamx.version>
+    <streamx.version>1.0.0</streamx.version>
```
Comment (michalcukierman) [inline pom.xml:42]: I did that for you ;)

Comment (marekczajkowski) [reply pom.xml:42]: Thanks :)

**Review by marekczajkowski — APPROVED** [review]

**Pull request merged**